src/core.js

```diff
@@ -2186,8 +2186,13 @@ define([
             success: function (data) {
                 if (saveType === 'patch') {
                     if (data.status === 'conflict') {
-                        if (_.isUndefined(data.xform)) {
+                        // reset save button to unsaved state
+                        _this.data.core.saveButton.fire("change");
+                        var force_full = _this.opts()
+                            .features.full_save_on_missing_conflict_xform;
```

Comment:
Does adding this mean that user's can no longer save on conflicting changes? I don't see an HQ PR to set this feature

Reply:
No, it's a feature meant to be used in vellum (standalone mode) only. It's set in `tests/options.js` and not intended to be used anywhere else. Basically it just ensures that the branch with the comment `unconditionally overwrite if no xform to compare` is never executed in production.